gateway/src/main/java/com/mx/path/gateway/configuration/AccessorConstructionContext.java

```diff
@@ -181,8 +175,7 @@ private <K extends T> Constructor<K> findBestConstructor(Class<K> klass) {
         c -> Arrays.stream(c.getParameters()).allMatch(
             param -> param.getType() == AccessorConfiguration.class
                 || param.isAnnotationPresent(Configuration.class)
-                || param.isAnnotationPresent(Connection.class)
-                || param.isAnnotationPresent(ClientID.class)))
+                || param.isAnnotationPresent(Connection.class)))
```

Comment:
Why this change?

Reply:
I'm removing ClientID annotated parameters as valid from accessor constructors. Isn't that what we decided?

Reply:
Got it.